client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html

```diff
@@ -488,6 +488,23 @@
                         </span>
                     </div>
 
+                    <!-- ADDRESS EXTRA-->
+                    <div class="form-group">
+                      <label for="contactAddressExtra"
```

Comment:
can you please use 4 spaces for indent ?

Reply:
`data-translate` could be directly on the `<label>` element.